packages/administration/src/Component/Config/Action/Builder/AbstractAction.php

```diff
@@ -32,6 +28,17 @@ abstract class AbstractAction
      */
     abstract public static function create(string $name, string $label): self;
 
+    /**
+     * @return string
+     */
+    abstract protected function getTemplate(): string;
+
+    /**
+     * @param object|null $entity
+     * @return array<string, mixed>
+     */
+    abstract protected function getTemplateParameters(?object $entity): array;
```

Comment:
is $entity a proper name?

Reply:
or otherwise, should it be named `getTemplateParameters` when you need to pass the entity?

Reply:
Do you have any suggestions, on how to avoid passing that parameter? In the other hand, it still says what it does. Like you will get TemplateParameters

Reply:
yeah, my bad, I probably misunderstood the method in the first place. However, now, when I see the usage, what about this?

![image](https://github.com/user-attachments/assets/c9075e22-13ae-4e0b-91c9-1431e6f42279)